homeassistant/components/lock/nuki.py

```diff
@@ -80,19 +79,18 @@ def service_handler(service):
         path.join(path.dirname(__file__), 'services.yaml'))
 
     hass.services.register(
-        DOMAIN, SERVICE_LOCK_N_GO, service_handler,
+        NUKI_DATA, SERVICE_LOCK_N_GO, service_handler,
```

Comment:
No, services are registered under the base component domain, even from the platforms. So use `DOMAIN` here.